examples/using_smirnoff_with_amber_protein_forcefield/toluene_in_T4_lysozyme.ipynb

```diff
@@ -95,29 +97,30 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "t4_pdb_file_path = get_data_filename('proteins/T4-protein.pdb')\n",
-    "t4_pdbfile = PDBFile(t4_pdb_file_path)\n",
-    "\n",
-    "# Load the AMBER protein force field through OpenMM.\n",
+    "# Load the AMBER protein force field parameters through OpenMM.\n",
     "from simtk.openmm import app\n",
-    "omm_forcefield = app.ForceField('amber99sbildn.xml')\n",
+    "omm_forcefield = app.ForceField('amber99sbildn.xml', 'tip3p.xml')\n",
```

Comment:
@jchodera At the January meeting, [there seemed to be consensus](https://docs.google.com/presentation/d/1R2zM1bTqqDHUC534UKdI8vbdov2peNrQ3lA5-wtX_bc/edit#slide=id.g4d411744a8_0_0) that we would "pin" our small molecule forcefield to proteins parameterized using [AMBER ff14-sb](https://github.com/pandegroup/openmm/blob/master/wrappers/python/simtk/openmm/app/data/amber14/protein.ff14SB.xml) and water using [TIP3P-FB](https://github.com/pandegroup/openmm/blob/master/wrappers/python/simtk/openmm/app/data/tip3pfb.xml). Do we want to start putting that in our examples?

Reply:
Good idea! I'll swap the models before merging.